docs/_schema_tables/AlternateName.csv

```diff
@@ -1,5 +1,5 @@
 Field Name,Description,Format
-type,"Type: What kind of alternative name is this? Select from 'translation', 'formerName','birth', and 'alias'. List options: [ 'translation', 'former', 'alias', 'birth' ]",string
+type,"Type: What kind of alternative name is this? Select from 'translation', 'formerName','birth', and 'alias'.",string

```

Comment:
I think this is a typo in the script that generates the front-end view. Will get that fixed on master.